templates/home/index.html.twig

```diff
@@ -1,7 +1,56 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %} Projektai {% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
-{% endblock %}
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for key, team in data %}
+            {% for student in team.students %}
+                <li class="list-group-item">
+                    <a href="{{ path('student', {'name': student , 'project': key })}}">{{ student }}</a>
+                    (
+                    <span class="badge">Mentorius</span> {{ team.mentors.0 }}
+                    )
+                </li>
+            {% endfor %}
+        {% endfor %}
+    </ul>
+
+    <div class="m-4">
+        <form action="{{ path('student') }}" method="get">
+            <div class="input-group">
+                <input name="name" type="text" class="form-control" placeholder="Studentas"/>
+            </div>
+            <div class="input-group">
+                <input name="project" type="text" class="form-control" placeholder="Projektas"/>
+            </div>
+            <button type="submit" class="btn btn-success">Sužinoti vertinimą</button>
+        </form>
+    </div>
+
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Projektai</li>
+        {% for key, team in data %}
+            <li class="list-group-item">
+                <h3>{{ key }}</h3>
+                <div class="panel panel-default">
+                    <div>
+                        <a target="_blank" href="https://github.com/nfqakademija/{{ team.name | url_encode }}">github.com/nfqakademija/{{ key }}</a>
```

Comment:
Čia naudojami ne tie duomenys:
![image](https://user-images.githubusercontent.com/1453957/70394061-c7ed7300-19f9-11ea-88c1-632eacb4388e.png)
:vs: 
![image](https://user-images.githubusercontent.com/1453957/70394069-dfc4f700-19f9-11ea-8cbb-10c2cc1eda54.png)

Todėl testai neprasisuka:
https://github.com/aurelijusb/kickstart/pull/198/checks?check_run_id=337412662
![image](https://user-images.githubusercontent.com/1453957/70394079-fcf9c580-19f9-11ea-961e-3b3fd7437a6e.png)
![image](https://user-images.githubusercontent.com/1453957/70394090-1ef34800-19fa-11ea-9a13-a39594946c0d.png)

Tu jau naudoji `key, team`. Tai palygink, kokia yra reikšmė yra `key` :vs: `team.name.

Plačiau:
 * https://twig.symfony.com/doc/2.x/tags/for.html#iterating-over-keys-and-values
 * https://www.php.net/manual/en/control-structures.foreach.php